examples/customize-rows/index.js

```diff
@@ -0,0 +1,96 @@
+/*
+  See another example of how to use `customRowRender` at
+  https://github.com/Skn0tt/mui-datatables-responsive-demo
+  https://mui-datatables-responsive-demo.skn0tt.now.sh
+*/
+
+import React from 'react';
+import ReactDOM from "react-dom";
+import MuiDatatable from "../../src/";
```

Comment:
For consistency, we should keep the case the same as it is elsewhere, so it would `MuiDataTable` instead of `MuiDatatable`.

Reply:
Fixed in https://github.com/gregnb/mui-datatables/pull/589/commits/c57a81c6cc47652231b1ef20294147c5c221e460.